From d44b005c03d21e6f205856c2bd7bb9c2ceedf0af Mon Sep 17 00:00:00 2001
From: Yanting Yang <yantingyang@google.com>
Date: Thu, 14 Oct 2021 15:06:33 +0000
Subject: [PATCH] Rephrase dialog message of clear storage dialog for security
 concern

Bug: 193890833
Test: visual
Change-Id: I8f0b066de710169ee8b922c44b6519ca21b9c7ef
(cherry picked from commit 0c359da620498d536d81cb97d5ae48048201c226)
(cherry picked from commit 9a0c28d911046e7cbfdfc8cce1fab94394588c51)
Merged-In:I8f0b066de710169ee8b922c44b6519ca21b9c7ef
---
 res/values/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values/strings.xml b/res/values/strings.xml
index c4609fb1f5..133b98392b 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -4813,7 +4813,7 @@
     <!-- Manage applications, individual application screen, confirmation dialog title. Displays when user selects to "Clear data". -->
     <string name="clear_data_dlg_title">Delete app data?</string>
     <!-- Manage applications, individual application screen, confirmation dialog message. Displays when user selects to "Clear data". It warns the user of the consequences of clearing the data for an app. -->
-    <string name="clear_data_dlg_text">All this app\u2019s data will be deleted permanently. This includes all files, settings, accounts, databases, etc.</string>
+    <string name="clear_data_dlg_text">This app\u2019s data will be permanently deleted. This includes files, settings, databases, and other app data.</string>
     <!-- Manage applications, individual application screen, confirmation dialog button. Displays when user selects to "Clear data". Goes through with the clearing of the data. -->
     <string name="dlg_ok">OK</string>
     <!-- Manage applications, individual application screen, confirmation dialog button. Displays when user selects to "Clear data". -->
-- 
2.17.1

